srcpkgs/tflint/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'tflint'
 pkgname=tflint
-version=0.22.0
+version=0.45.0
 revision=1
 build_style=go
 go_import_path="github.com/terraform-linters/$pkgname"
 short_desc="Terraform linter focused on possible errors, best practices, etc"
-maintainer="Sora Morimoto <sora@morimoto.io>"
+maintainer="Emil Tomczyk <emil.tomczyk@skni.umcs.pl>"
```

Comment:
@smorimoto can they adopt this from you?

Reply:
I don't have any problem with it! Rather, I'm sorry that I haven't been able to take care of this for a long time! 😱